recipe/meta.yaml

```diff
@@ -17,6 +17,11 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - meson-python                           # [build_platform != target_platform]
+    - ninja                                  # [build_platform != target_platform]
```

Comment:
@isuruf I feel like I'm missing something obvious here. I added the last two and got a bit further (or just a different error but I chose to look at the glass half full ;-p)

Do you mind taking a quick look if we can or can't do osx-arm64 for this fortran extension?

Reply:
Probably need to pass MESON_ARGS. See https://github.com/conda-forge/numpy-feedstock/blob/main/recipe/build.sh